src/python/doc/rips_complex_tflow_itf_ref.rst

```diff
@@ -0,0 +1,39 @@
+:orphan:
+
+.. To get rid of WARNING: document isn't included in any toctree
+
+TensorFlow layer for Vietoris-Rips persistence
+##############################################
+
+.. include:: differentiation_sum.inc
+
+Example of gradient computed from Vietoris-Rips persistence
+-----------------------------------------------------------
+
+.. testcode::
+
+    from gudhi.tensorflow import *
```

Comment:
Maybe to be reworked according to [this discussion](https://github.com/GUDHI/gudhi-devel/pull/411#discussion_r742337781)
```suggestion
    from gudhi.tensorflow import RipsLayer
```

Reply:
Thanks! Done.